modules/ROOT/pages/policy-visualization.adoc

```diff
@@ -0,0 +1,51 @@
+= Policy Visualization
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== About the Policy Visualization
+
+[NOTE]
+When deploying a Mule app, you must ensure that you are using a version of Mule that supports the Policy Visualization. Learn more about which xref:setup.adoc#select-a-deployment-type[runtimes are supported].
+
+The Policy Visualization enables you to see the relationship between Mule app deployments and the APIs they are implementing and which policies have been applied to those APIs. Data about which policies have been applied to your APIs comes from API Manager.
```

Comment:
I stumbled over the distance between "to see the relationship..." and "(to see) which policies." Maybe a comma?
```suggestion
The Policy Visualization feature enables you to see the relationship between Mule app deployments and the APIs they are implementing, and which policies are applied to those APIs. Data about which policies are applied to your APIs comes from API Manager.
```
It still sounds a bit odd. I wonder if we should make the "which policies" part a separate sentence?